src/interfaces/IWeather.tsx

```diff
@@ -0,0 +1,6 @@
+export interface IWeather {
```

Comment:
Should be using `type` rather than `interface`. 

We would typically use an `interface` to reduce repeated code if there are two objects that share similar properties.

See example below
```Typescript
interface Animal {
  name: string,
  color: string,
}

interface Snake extends Animal {
  length: number,
}

const snake: Snake = getSnake()
snake.name
snake.length
```